### ISSUE TITLE
Doors

### ISSUE THREAD

**Issue by TheAmazingSpy** [Electrical-Age/ElectricalAge#328]
I have some idea for doors, and this does depend on how good your animation is, but here it goes,
1. Fire Door w/ alarm: Basically, what they do is when opened, it can send a signal through a cable from ABOVE the door which you can do whatever with, just a small novelty feature but very cool in my eyes.
2. Sliding door: This door can be opened with a signal OR with an idea called the 'compact player sensor' that sensors UNDER the block, and no need for wires (unless you are wiring to something else) and it can come in different designs i.e: glass, metal, etc
3. Heavy Door: This is an EXPLOSION PROOF door, which uses electricity to open, not a signal, and I think it should be 3x5 or 2x4 and slides DOWN and UP, and I think it should use a small amount to open.
I have pictures for each of the doors.
4. Vault Door: This door will be openable with a signal, and for now, it can be a simple animation but my envision is that some gears turn and things like that and voila it's opened
![vault-door-knob](https://cloud.githubusercontent.com/assets/13247665/11605738/53ce0186-9b5b-11e5-9325-f1bccbc2540d.jpg)

![fire-door-11](https://cloud.githubusercontent.com/assets/13247665/11605723/c160a6f0-9b5a-11e5-96b1-84bb12ebf858.jpg)
![stylish-electric-sliding-screen-door-idea](https://cloud.githubusercontent.com/assets/13247665/11605727/e82b96fa-9b5a-11e5-9af8-41adad74db14.jpg)
![the_heavy_metal_door_by_hossein_mahmoodi-d57wnoh](https://cloud.githubusercontent.com/assets/13247665/11605732/0b4b32da-9b5b-11e5-94ff-2c6bdf627a11.jpg)

**Comment by Baughn** [comment]
The ideas are somewhat interesting, but do you have blender models for any of this? It'd help.

**Comment by TheAmazingSpy** [comment]
Nope, I am on a mac and don't know how to model, but I could learn or otherwise, yeah

**Comment by TheAmazingSpy** [comment]
I could make a visual representation in craft studio, but then I can't export it but a screenshot should do

**Comment by TheAmazingSpy** [comment]
Would you consider adding to 'Electrical Doors' for that milestone?

**Comment by TheAmazingSpy** [comment]
Okay, fire door: 
![screen shot 2015-12-06 at 6 27 58 pm](https://cloud.githubusercontent.com/assets/13247665/11612271/345d5fda-9c47-11e5-910a-cc2f4c782fa5.png)
![screen shot 2015-12-06 at 6 28 13 pm](https://cloud.githubusercontent.com/assets/13247665/11612275/5004ebb8-9c47-11e5-9c7d-fd3b6ad67e1f.png)

**Comment by TheAmazingSpy** [comment]
Really basic, but anyone good at blender, please if you have some time make a better one
